src/app/components/Results/Results.jsx

```diff
@@ -48,17 +48,17 @@ class Results extends React.Component {
 
   getResultsSort() {
     return sortingOpts.map((d, i) => {
-      return (<li role="region" key={i} onClick={e => this.sortResultsBy(e, d.val, d.label)}>
-        {d.label}
+      return (<li role="region" key={i}>
+        <a href="#" onClick={e => this.sortResultsBy(e, d.val, d.label)}>{d.label}</a>
```

Comment:
Ahhhhhhh. I made listeners. This is better, but maybe @EdwinGuzman knows a reason why it should be done the other way (or maybe I just did it that way 'cuz 🙃 ).